falcon/request.py

```diff
@@ -2090,7 +2093,7 @@ class RequestOptions:
     auto_parse_qs_csv: bool
     strip_url_path_trailing_slash: bool
     default_media_type: str
-    media_handlers: Handlers
+    media_handlers: UserDict
```

Comment:
Is there any reason why this is a blank `UserDict` whereas `Response.media_handlers` is typed as `falcon.typing.MediaHandlers`?